pep-0517.txt

```diff
@@ -480,6 +482,13 @@ Integration frontends require that an sdist named
 ``{NAME}-{VERSION}.{EXT}`` will generate a wheel named
 ``{NAME}-{VERSION}-{COMPAT-INFO}.whl``.
 
+The new restrictions for sdists built by PEP 517 backends are:
+
+- They will be gzipped tar archives, with the ``.tar.gz`` extension. Zip
+  archives, or other compression formats for tarballs, are not allowed at
+  present.
+- Tar archives must be created in the modern POSIX.1-2001 pax tar format, which
+  uses UTF-8 for file names.
```

Comment:
We also expect PEP 517 sdists to contain a `pyproject.toml` file that defines `build-system.build-backend` (otherwise they'll be handled as legacy sdists and expected to contain a `setup.py` file).